main.js

```diff
@@ -0,0 +1,177 @@
+                        '2100 Highest Observed Water (HOW) Mean',
+                        '2050 Highest Observed Water (HOW) Max',
+                        '2050 Highest Observed Water (HOW) Mean',
+                        '2010 Highest Observed Water (HOW)'
+                    ],
+                    profile = {
+                        how:  { mean: [0, 8, 6], max: [0, 7, 5] },
+                        mhhw: { mean: [4, 3, 1], max: [4, 2, 0] }
+                    },
+                    layerIds = profile[tidal][sea],
+                    labels = layerIds.map(function(id) { return desc[id]; }),
+                    baseUrl = 'http://dev.services.coastalresilience.org/arcgis/rest/services/Puget_Sound/Nisqually_Flood_and_SLR/MapServer/';
+
+                this.animateLayers(this.locations.nisqually, baseUrl, layerIds, labels, $stepLabel);
+            },
+
+            animateLayers: function(location, baseUrl, layerIds, labels, $display) {
+                var self = this,
+                    step = 0,
+                    pauseTime = 6000,
+                    showLayer = function(id) {
+                        return function() {
+                            var newLayer = new MapImageLayer({
+                                url: baseUrl,
+                                sublayers: [{id: id}]
+                            });
+
+                            transitionLayers(self.currentLayer, newLayer);
+                            $display.text(labels[step++]);
+                        };
+                    },
+                    transitionLayers= function(oldLayer, newLayer) {
+                            self.map.addLayer(newLayer);
+                            setTimeout(function() {
+                                removeLayer(oldLayer);
+                                // Keep a reference to the most recent layer
+                                self.currentLayer = newLayer;
+                            }, 1500);
+                    },
+                    removeLayer = function(layer) {
+                        if (layer) {
+                            self.map.removeLayer(layer);
+                        }
+                    },
+                    pause = function(time) {
+                        var defer = $.Deferred();
+                        return function() {
+                            setTimeout(function() {
+                                defer.resolve();
+                            }, time);
+                            return defer;
+                        };
+                    };
+
+                $display.empty();
+                removeLayer(self.currentLayer);
+                var animationStep = this.zoomTo(location);
+                layerIds.forEach(function(id) {
+                    animationStep = animationStep
+                        .then(showLayer(id))
+                        .then(pause(pauseTime));
+                });
+                return animationStep;
```

Comment:
Consider disabling the "Show Flood Layers" button while the animation is in progress, and maybe adding some kind of "loading" indicator? The layers don't load particularly fast, so having more visual cues could be helpful.